src/mappings/types/identity.ts

```diff
@@ -0,0 +1,12 @@
+export interface IdentityUpsertData {
```

Comment:
why is it called `types` but `interface` is used?

Reply:
good thread https://stackoverflow.com/questions/37233735/interfaces-vs-types-in-typescript/52682220#52682220

especially the post "Relevant in 2021" regarding primitves, tuples etc. i think types are more consistent and the merging feature of interface is causing less concise code,.